src/components/PopupWithForm.js

```diff
@@ -15,19 +30,42 @@ export default class PopupWithForm extends Popup {
     });
     return inputValues;
   }
-  // close() {
-  //   this._popupForm.reset();
-  //   super.close();
-  // }
+
+  open() {
+    this._popupForm.reset();
```

Comment:
You should **clear ** the inputs only after **submitting** a card in `then` rather than when you open or close popups because if you unexpectedly close the popup you will be upset if the inputs are empty when you open it again and you’ll have to type the data again.